rskj-core/src/main/java/co/rsk/peg/union/constants/UnionBridgeRegTestConstants.java

```diff
@@ -27,34 +28,40 @@ private UnionBridgeRegTestConstants() {
         lockingCapIncrementsMultiplier = 4;
 
         // TODO: Replace with actual authorizers
-        List<ECKey> changeUnionBridgeContractAddressAuthorizers = Collections.singletonList(
-            // seed: unionBridgeAuthorizer
-            ECKey.fromPublicOnly(Hex.decode(
-                "041fb6d4b421bb14d95b6fb79823d45b777f0e8fd07fe18d0940c0c113d9667911e354d4e8c8073f198d7ae5867d86e3068caff4f6bd7bffccc6757a3d7ee8024a"))
-        );
+        // seed: unionBridgeAuthorizer
+        List<ECKey> changeUnionBridgeContractAddressAuthorizers = Stream.of(
```

Comment:
No need to be a list of it's a single element, right?

Reply:
Fixed